app/build.gradle

```diff
@@ -48,11 +48,13 @@ dependencies {
     implementation "androidx.lifecycle:lifecycle-extensions:$Versions.lifecycleEx"
     implementation "androidx.paging:paging-runtime:$Versions.paging"
     implementation "androidx.recyclerview:recyclerview:$Versions.androidX"
-    implementation 'com.google.android.material:material:1.1.0-alpha03'
+    implementation "com.google.android.material:material:$Versions.androidX"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/worker8/Pixels/blob/814cae669abc38d21778fc91279b72bfb75a1599/app/build.gradle#L51"></span>A newer version of com.google.android.material:material than 1.1.0-alpha02 is available: 1.1.0-alpha03</td>
    </tr>
  </tbody>
</table>